src/models/sql.py

```diff
@@ -131,12 +125,14 @@ def import_from_local(self, cur, cleaned):
     def load_tables_from_public(self, cur):
         created_tables = []
 
+        parse_catalog_clause = " or table_schema = 'pg_catalog'" if self.config.model_config.parse_catalog else ""
```

Comment:
Naming the flag as `load-catalog-tables`,  `load-catalog-table-metadata`, etc. may make more sense than `parse-catalog`.